sklearn/model_selection/__init__.py

```diff
@@ -0,0 +1,50 @@
+from .split import BaseCrossValidator
+from .split import KFold
+from .split import StratifiedKFold
+from .split import LeaveOneLabelOut
+from .split import LeaveOneOut
+from .split import LeavePLabelOut
+from .split import LeavePOut
+from .split import ShuffleSplit
+from .split import StratifiedShuffleSplit
+from .split import PredefinedSplit
+from .split import train_test_split
+from .split import check_cv
+
+from .validate import cross_val_score
+from .validate import cross_val_predict
+from .validate import learning_curve
+from .validate import permutation_test_score
+from .validate import validation_curve
+
+from .search import GridSearchCV
+from .search import RandomizedSearchCV
+from .search import ParameterGrid
+from .search import ParameterSampler
+from .search import fit_grid_point
+
+__all__ = ('split',
```

Comment:
I am -0 on putting each on it's own line. But if you do, maybe order them alphabetically?